etc/helm/pachyderm/templates/console/secret.yaml

```diff
@@ -12,9 +12,6 @@ metadata:
   name: pachyderm-console-secret
   namespace: {{ .Release.Namespace }}
 data:
-{{ if .Release.IsUpgrade }}
-  OAUTH_CLIENT_SECRET: {{ required "For an upgrade release, a value is required for console.config.oauthClientSecret" .Values.console.config.oauthClientSecret | toString | b64enc | quote }}
-{{ else }}
+  OAUTH_CLIENT_ID: {{ .Values.console.config.oauthClientID | b64enc | quote }}
```

Comment:
this may be a backwards compatability issue where we actually want to have them add this to their secret pre-upgrade